### PR TITLE
Automated PR for CATOSL-1.1

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/CATOSL-1.1.xml [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+         growth, and innovation. To demonstrate our continuing commitment to open source, we are releasing the
+         Program (as defined below) under this License.</p>
+      <p>This License is intended to permit contributors and recipients of the Program to use the Program,
+         including its source code, freely and without many of the concerns of some other open source licenses.
+         Although we expect the underlying Program, and Contributions (as defined below) made to such Program,
+         to remain open, this License is designed to permit you to maintain your own software programs free of
+         this License unless you choose to do so. Thus, only your Contributions to the Program must be
+         distributed under the terms of this License.</p>
+      <p>The provisions that follow set forth the terms and conditions under which you may use the Program.</p>
+    </optional>
```
Comment (jlovejoy) [inline src/CATOSL-1.1.xml]: we didn't have this as optional before - not sure it should be now ??

File: src/CATOSL-1.1.xml [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
+        <list>
+          <li>
+            <b>*</b>
+            <p>it complies with the terms and conditions of this License; and</p>
+          </li>
+          <li>
+            <b>*</b>
+            <p>its license agreement:</p>
+          </li>
+          <list>
```
Comment (jlovejoy) [inline src/CATOSL-1.1.xml]: extra list tag here?

File: src/CATOSL-1.1.xml [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+              <b>*</b>
+              <p>states that any provisions which are inconsistent with this License are offered by that
+                 Contributor alone and not by any other party; and</p>
+            </li>
+            <li>
+              <b>*</b>
+              <p>states that source code for the Program is available from such Contributor at the cost of
+                 distribution, and informs licensees how to obtain it in a reasonable manner.</p>
+            </li>
+          </list>
```
Comment (jlovejoy) [inline src/CATOSL-1.1.xml]: extra end list tag

**Comment by jlovejoy** [comment]
might be nice to add indented sub-lists for subsections (e.g., 1.2, 1.3, etc.) - this was not done in the original, but would look nicer...

File: src/CATOSL-1.1.xml (hunk 1 above)
Comment (bradleeedmondson) [inline src/CATOSL-1.1.xml]: agree; this (lines 11-28) should not be optional

**Comment by jlovejoy** [comment]
I updated list nesting... @myndzi - can you check it?

**Comment by myndzi** [comment]
Was a missing </li>, fixed it and something else, no warnings for xml structure now

File: src/CATOSL-1.1.xml (hunk 1 above)
Comment (pmadick) [inline src/CATOSL-1.1.xml]: agreed.  Fix and submit.

**Pull request merged**